tests/helper/helper_filesystem.go

```diff
@@ -296,3 +296,15 @@ func ReadFile(filePath string) (string, error) {
 	}
 	return string(data), nil
 }
+
+// CreateSimpleFile creates a simple file
+// return the file path with random string
+func CreateSimpleFile(context, filePrefix, fileExtension string) (string, string) {
+
+	FilePath := filepath.Join(context, filePrefix+RandString(10)+fileExtension)
+	content := []byte(RandString(10))
+	err := ioutil.WriteFile(FilePath, content, 0644)
```

Comment:
`0644` is throwing security issues https://travis-ci.com/github/openshift/odo/jobs/393570498#L1197

Reply:
Fixed in https://github.com/openshift/odo/pull/4067/commits/99647241a752fac1029150e953299e6c55d9d9a3